scp.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	scp "github.com/bramvdbogaerde/go-scp"
+	"github.com/bramvdbogaerde/go-scp/auth"
+	"golang.org/x/crypto/ssh"
+	"os"
+	"time"
+)
+
+func scpToWorker(source string, dest string, tarName string) error {
+	// Use SSH key authentication from the auth package
+	// we ignore the host key in this example, please change this if you use this library
+	clientConfig, err := auth.PrivateKey("ubuntu", "/home/ubuntu/.ssh/senior-design.pem", ssh.InsecureIgnoreHostKey())
+	if err != nil {
+		Error.Println("Error creating ssh config", err)
+		return err
+	}
+	// For other authentication methods see ssh.ClientConfig and ssh.AuthMethod
+
+	// Create a new SCP client
+	client := scp.NewClientWithTimeout("ec2-54-211-200-158.compute-1.amazonaws.com:22", &clientConfig, time.Duration(100000000000))
+
+	// Connect to the remote server
+	Info.Println("Connecting to worker...")
+	err = client.Connect()
+	if err != nil {
+		Error.Println("Couldn't establish a connection to the remote server ", err)
+		return err
+	}
+
+	// Open a file
+	f, err := os.Open(source)
+	if err != nil {
+		Error.Println("Error opening source file scp", err)
+		return err
+	}
+
+	// Close client connection after the file has been copied
+	defer client.Close()
+
+	// Close the file after it has been copied
+	defer f.Close()
+
+	// Finaly, copy the file over
+	// Usage: CopyFile(fileReader, remotePath, permission)
+	Info.Println("Copying " + tarName)
+	return client.CopyFile(f, dest, "0655")
```

Comment:
Why `0655`? Needs a comment

Reply:
https://github.com/bramvdbogaerde/go-scp
This is the octal used in the example from the scp package. I can change it to something different if you want. Or maybe comment a small description of the permissions.

Reply:
`0664` please, and add a comment that it means: read/write for owner/group, and then read only for everyone else